app/javascript/src/components/ChartCard/BarChartByTechnicalArea.js

```diff
@@ -121,6 +131,7 @@ class BarChartByTechnicalArea extends React.Component {
       objOfActionCounts
     )
     const stackedBarEls = [$elBarSegmentA, $elBarSegmentB]
+    $(".tooltip").remove()
```

Comment:
This may be problematic and there may be a smarter way to do this, as discussed "in person".

Reply:
fixed